chore(curriculum): add newspaper article lab (#55844)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md

---
id: 66ba762af611169359d9d369
title: Build a Newspaper Article
challengeType: 14
dashedName: build-a-newspaper-article
demoType: onClick
---

# --description--

**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 

**User Stories:**

1. You should set the root `font-size` of your HTML document to `24px`.

> 1. You should have an element with a class of `newspaper` that contains all your other elements.

Reviewer: ```suggestion
1. You should have an element with a class of `.newspaper` that contains all your other elements.
```

1. Your `.newspaper` element should have a `font-size` of `16px` and a font of `Open Sans` with a fallback font of `sans-serif`.

> 1. Within your `.newspaper` element, you should have at least seven more elements: one for the newspaper name that has a class of `name`, one for the the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`. All of these elements should be filled with your article information.

Reviewer: ```suggestion
1. Within your `.newspaper` element, you should have at least seven more elements: one for the newspaper name that has a class of `name`, one for the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`. All of these elements should be filled with your article information.
```

> 1. Your `.name` element should have a `font-size` that's two times the size of the root elements `font-size` and a font of `Times New Roman` with a fallback font of `serif`.

Reviewer: ```suggestion
1. Your `.name` element should have a `font-size` that is twice the root element's `font-size` and should use the `Times New Roman` font, with a fallback font of `serif`.
```

> 1. Your `.name` and `.author` elements should use CSS to make all its characters uppercase.

Reviewer: ```suggestion
1. Your `.name` and `.author` elements should use CSS to make all their characters uppercase.
```

> 1. Your `.headline` element should `font-size` that's two times the size of its parent's `font-size` and use CSS to make it bold.

Reviewer: ```suggestion
1. Your `.headline` element should have a `font-size` that is twice its parent element's `font-size` and should be bold.
```

> 1. Your `.sub-headline` element should have a `font-weight` of `100`, a `font-size` that's `1.5` times the size of its parent's `font-size`, and use CSS to make it italicized.

Reviewer: ```suggestion
1. Your `.sub-headline` element should have a `font-weight` of `100`, a `font-size` that is `1.5` times its parent element's `font-size`, and should be italicized.
```

1. Your `.author` should use CSS to make it bold.

> 1. Your `.text` elements should be indented `20px`.

Reviewer: ```suggestion
1. Your `.text` elements should have a `text-indent` of `20px`.
```

> 1. Your `.text` elements should have a `line-height` that's two times the size of its parent's `font-size`.

Reviewer: ```suggestion
1. Your `.text` elements should have a `line-height` that is twice their parent element's `font-size`.
```

> 1. The first letter of your `.text` elements should be bold and two times the size of its parent's `font-size`. Hint: use the `::first-letter` selector.

Reviewer: ```suggestion
1. The first letter of your `.text` elements should be bold and twice the size of their parent element's `font-size`. Use the `::first-letter` selector for this.
```
